services/common/rs_server_common/authentication.py

```diff
@@ -134,19 +157,31 @@ def apikey_validator(station, access_type):
     def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
-            if settings.cluster_mode():
+            if settings.CLUSTER_MODE:
```

Comment:
@agrosu1978 here I've modified your code to implement a specific error message when the cadip station is unknown, let me know if it's ok for you.